### PR TITLE
Ask for an organsation when requesting an account

### PR DESCRIPTION
**Pull request by cbaines**
Then provide programme options based on this selection.

image

## Programme page when Government Digital Service is selected
image

## Programme page when an Other organisation is selected
image

### CODE REVIEW

**Comment by Krenair** [comment]
As mentioned in DM, let's refer to the CDIO's directorate as `Cabinet Office Digital and Information Office` until it gets its own name?
GaaP should probably have a capital P too

**Comment by Krenair** [comment]
Also I think we have a file missing - `ActionView::Template::Error: uninitialized constant ProgrammeController::OrganisationController`

**Comment by cbaines** [comment]
> As mentioned in DM, let's refer to the CDIO's directorate as `Cabinet Office Digital and Information Office` until it gets its own name?

I think that sounds better, but here is probably the wrong place to try and effect a name change. I was going off https://gds-way.cloudapps.digital/standards/cdio-pillars.html#chief-digital-and-information-office-teams which says "Chief Digital and Information Office".

> GaaP should probably have a capital P too

Updated.

> Also I think we have a file missing - ActionView::Template::Error: uninitialized constant ProgrammeController::OrganisationController

and yeah, I just forgot to stage a load of files... they've been added now.

File: app/controllers/organisation_controller.rb [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        'GOV.UK',
+        'GOV.UK Verify',
+        'Government as a Platform',
+      ],
+    },
+    'Chief Digital and Information Office' => {
+      'programmes' => [
+        'Security',
+        'GovWifi',
+        'Digital Marketplace',
```
[the remark this answers is not in the document]
Comment (cbaines) [reply app/controllers/organisation_controller.rb:14]: I've added Digital Services now :+1:

**Review by Krenair — APPROVED** [review]

**Pull request merged**